src/terminal/adapter/adaptDispatchGraphics.cpp

```diff
@@ -12,6 +12,19 @@
 using namespace Microsoft::Console::VirtualTerminal;
 using namespace Microsoft::Console::VirtualTerminal::DispatchTypes;
 
+// Inspired from RETURN_IF_WIN32_BOOL_FALSE

```

Comment:
This should go into a more public helper section than just the header of this file and also link to an upstream contribution to Microsoft/wil if you've indeed identified a gap here.

Reply:
I've filed microsoft/wil#98 upstream to discuss. I'll move this somewhere more general